<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>中心</string>
  </property>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>161</x>
     <y>15</y>
     <width>781</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>779</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>建立连接</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="serverStatusLabel">
          <property name="font">
           <font>
            <family>方正黑体</family>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="text">
           <string>等待监听。。</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="serverProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="port">
          <property name="text">
           <string>99999</string>
          </property>
          <property name="placeholderText">
           <string>端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>开始监听</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>发送消息</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>779</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>监控模块</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>681</width>
       <height>361</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>系统信息</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>661</width>
         <height>321</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>进程信息</string>
      </attribute>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>571</width>
         <height>311</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>PID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>状态</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>优先级</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>占用内存</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>50</y>
         <width>83</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>杀掉此进程</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>模块信息</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_2">
         <column>
          <property name="text">
           <string>模块名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>占用内存</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>使用次数</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>资源信息</string>
      </attribute>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>40</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>占用率：</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>84</x>
         <y>9</y>
         <width>251</width>
         <height>25</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="outlabel_2">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>240</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>332</x>
         <y>240</y>
         <width>16</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>共</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>30</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>已用</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>240</y>
         <width>30</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>已用</string>
       </property>
      </widget>
      <widget class="QLabel" name="inlabel_2">
       <property name="geometry">
        <rect>
         <x>195</x>
         <y>170</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="inlabel_1">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>170</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>69</width>
         <height>25</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>CPU占用</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>170</y>
         <width>30</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余</string>
       </property>
      </widget>
      <widget class="QLabel" name="inlabel_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>170</y>
         <width>121</width>
         <height>19</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>170</y>
         <width>16</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>共</string>
       </property>
      </widget>
      <widget class="QLabel" name="cpulabel">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>61</width>
         <height>19</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="outlabel_3">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>240</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="outlabel_1">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>240</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>157</x>
         <y>240</y>
         <width>30</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>剩余</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>90</y>
         <width>115</width>
         <height>25</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>内存和交换分区</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>208</y>
         <width>341</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>交换</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_3">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>131</y>
         <width>341</width>
         <height>27</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>内存</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar_2">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>779</width>
      <height>368</height>
     </rect>
    </property>
    <attribute name="label">
     <string>密钥分发模块</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>131</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
